evalml/automl/automl_search.py

```diff
@@ -631,6 +635,28 @@ def __init__(
         if not engine:
             self._engine = SequentialEngine()
         else:
+            valid_engines = [
+                "cf_threaded",
+                "cf_process",
+                "dask_threaded",
+                "dask_process",
+            ]
+            if isinstance(engine, str):
+                if engine in valid_engines:
+                    if engine == "cf_threaded":
+                        engine = CFEngine(CFClient(ThreadPoolExecutor()))
+                    elif engine == "cf_process":
+                        engine = CFEngine(CFClient(ProcessPoolExecutor()))
+                    elif engine == "dask_threaded":
+                        dd = import_or_raise("dask.distributed")
```

Comment:
I don't think we need `import_or_raise` since dask is listed as a core requirement.